.github/workflows/rust.yml

```diff
@@ -3,25 +3,13 @@ name: Rust
 
 on:
   push:
-    # Don't run tests, when only text files were modified
```

Comment:
This is the script that normally runs for pull request and commit messages. This should still ignore the listed paths, as we don't need to run the rest if nothing changed in a commit. Also to be kind to the GH admins ^^

Reply:
Ah, thanks! Reverted